### PR TITLE
k8s-tools: bump image

### PR DESCRIPTION
**Pull request by Halamix2**
**Description**

Changes proposed in this pull request:

- bump to newest k8s-tools image

**Related issue(s)**

### CODE REVIEW

File: resources/application-connector/values.yaml [hunk 1]
```diff
@@ -87,7 +87,7 @@ application_connectivity_certs_sync:
       namespace: istio-system
   image:
     repository: eu.gcr.io/kyma-project/tpi/k8s-tools
-    tag: "20210407-86c769bd"
```
Comment (franpog859) [inline resources/application-connector/values.yaml:90]: There is a bonus space. Will it be a problem?

**Review by majakurcius — APPROVED** [review]

File: resources/application-connector/values.yaml (hunk 1 above)
Comment (Halamix2) [reply resources/application-connector/values.yaml:90]: Thank you, fixed

**Review by suleymanakbas91 — APPROVED** [review]

**Review by marcobebway — APPROVED** [review]

**Review by adamwalach — APPROVED** [review]

**Review by colunira — APPROVED** [review]
/meow

**Review by franpog859 — APPROVED** [review]

**Review by tgorgol — APPROVED** [review]

**Review by majakurcius — APPROVED** [review]

**Pull request merged**